diff -urN openexr-1.4.0.org/IlmThread/Makefile.am openexr-1.4.0/IlmThread/Makefile.am
--- openexr-1.4.0.org/IlmThread/Makefile.am	2006-06-08 10:27:41.000000000 +0200
+++ openexr-1.4.0/IlmThread/Makefile.am	2007-01-02 22:05:05.844215500 +0100
@@ -11,7 +11,7 @@
 			  IlmThreadMutexPosix.cpp
 
 libIlmThread_la_LDFLAGS = -version-info @LIBTOOL_VERSION@ -no-undefined
-libIlmThread_la_LIBADD = ../Iex/libIex.la
+libIlmThread_la_LIBADD = ../Iex/libIex.la -lpthread
 
 libIlmThreadincludedir = $(includedir)/OpenEXR
 
@@ -23,4 +23,4 @@
 EXTRA_DIST = $(noinst_HEADERS) IlmThreadMutexWin32.cpp IlmThreadSemaphoreWin32.cpp \
 	     IlmThreadWin32.cpp
 
-INCLUDES = -I$(top_builddir) -I$(top_srcdir)/Iex -I$(top_srcdir)/config
\ Brak znaku nowej linii na końcu pliku
+INCLUDES = -I$(top_builddir) -I$(top_srcdir)/Iex -I$(top_srcdir)/config

