diff -urN OpenEXR-1.2.2.org/IlmImf/Makefile.am OpenEXR-1.2.2/IlmImf/Makefile.am
--- OpenEXR-1.2.2.org/IlmImf/Makefile.am	2004-12-17 03:01:57.000000000 +0100
+++ OpenEXR-1.2.2/IlmImf/Makefile.am	2006-07-30 19:47:39.140160250 +0200
@@ -2,6 +2,8 @@
 
 lib_LTLIBRARIES = libIlmImf.la
 
+libIlmImf_la_LIBADD = $(top_builddir)/Iex/libIex.la $(top_builddir)/Half/libHalf.la -lz
+
 libIlmImf_la_SOURCES = ImfAttribute.cpp ImfBoxAttribute.cpp ImfCRgbaFile.cpp \
 		       ImfChannelList.cpp ImfChannelListAttribute.cpp \
 		       ImfFloatAttribute.cpp ImfFrameBuffer.cpp \
diff -urN OpenEXR-1.2.2.org/Imath/Makefile.am OpenEXR-1.2.2/Imath/Makefile.am
--- OpenEXR-1.2.2.org/Imath/Makefile.am	2005-03-08 05:21:53.000000000 +0100
+++ OpenEXR-1.2.2/Imath/Makefile.am	2006-07-30 19:48:08.738010000 +0200
@@ -2,6 +2,8 @@
 
 lib_LTLIBRARIES = libImath.la
 
+libImath_la_LIBADD = $(top_builddir)/Iex/libIex.la
+
 libImath_la_SOURCES = ImathShear.cpp ImathMatrixAlgo.cpp ImathVec.cpp \
 		      ImathColorAlgo.cpp ImathFun.cpp \
 		      ImathColorAlgo.h ImathMatrixAlgo.h ImathVec.h \
